CMakeLists.txt

```diff
@@ -64,9 +64,9 @@ endif()
 
 # Use target ID syntax if supported for AMDGPU_TARGETS
 if(TARGET_ID_SUPPORT)
-  set(AMDGPU_TARGETS gfx900:xnack-;gfx906:xnack-;gfx908:xnack- CACHE STRING "List of specific machine types for library to target")
+  set(AMDGPU_TARGETS gfx803;gfx900:xnack-;gfx906:xnack-;gfx908:xnack- CACHE STRING "List of specific machine types for library to target")
```

Comment:
Please change this ```gfx803`` -> ```gfx803:xnack-```

Reply:
Thank you for responsing.
I followed other math projects like rocSPARSE, rccl. It left gfx803 without `xnack-`.

* https://github.com/ROCmSoftwarePlatform/rocSPARSE/commit/6cc282aaae2c28e880cf83e6934713282987d1a6
* https://github.com/ROCmSoftwarePlatform/rccl/commit/882d52ad7ec57e8055f71d978519c3352667fd5c

And <https://www.llvm.org/docs/AMDGPUUsage.html> said that gfx803 didnot support xnack, so it maybe neednot to config xnack for gfx803?

Reply:
@neon60 had changed `gfx803:xnack-`, please check.